examples/tomo/filtered_backprojection_helical_3d.py

```diff
@@ -17,29 +17,21 @@
 
 
 # Reconstruction space: discretized functions on the cube
```

Comment:
Actually, we could almost remove this whole line. Better version would be
`# Reconstruction space: functions on [-20, 20]^3, uniformly discretized with 200 samples per dimension`